### ISSUE TITLE
Streamed audio not playing.

### ISSUE THREAD

**Issue by Epicwade** [FIX94/Nintendont#1182]
TLDR: Streamed audio for this game works on swiss but not Nintendo so fix your stinky cringe baby mode
 audio implementation plz.

2021 or so
-JoshuaMK decompiles the audio streaming functionality from finding nemo on the GC.
-Implements that code into our custom sunshine game engine.
-noticed it only plays on dolphin.
-checks to see if it is his implementation is faulty or the audio so uses audio from finding nemo to test.
-spends next few monts reviewing and tweaking code and consulting other gc modders with no real conclusion.

Late January 2024
-A new rando joins the discord server for our mod.
-rando claims that streamed audio played on his GameCube
-I say "WFT you pulling my leg"
-He was not pulling my leg
-streamed audio works now on original GameCube hardware.
-it still doesn't work on Nintendont
-I make stinky issue request on github
-Please fix it before June if you can since this is gonna be one of the biggest GameCube mods out there and a lot of people will be dissappointed in the near future.

**Comment by madsl** [comment]
It doesn't help the cause if you are this toxic about a feature request. Who talks like this?

**Comment by Epicwade** [comment]
My intention was to be playfull not rood but would be cool if the code wasnt stinky is all. Im probably not the best person on the team to talk about coding stuff i mostly did object placement and 3D model work so do excuse me if I don't exactly have the vocabulary except calling it bad and stinky i don't know the fancy terms for coding stuff.
